src/ui/src/app/deployment-item/deployment-item.component.scss

```diff
@@ -0,0 +1,60 @@
+// Import your custom theme
+@import '../../theme.scss';
+
+.deployment_item {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  background: #EAEDEF;
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
+  border-radius: $border-radius;
+  margin: 1em;
+  color: inherit;
+  box-sizing: border-box;
+  overflow: hidden;
+  &_icon {
+    height: 100%;
+    width: 80px;
+    background: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    img {
+      width: 70%;
+      max-height: 100%;
+    }
+  }
+  &_content {
+    height: 100%;
+    position: relative;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    word-wrap: break-word;
+    overflow: hidden;
+    &__title {
+      margin: 0;
+      margin-bottom: 5px;
+      font-size: 1.2em;
+    }
+    &__subtitle {
+      font-size: 0.8em;
+      line-height: 1.5em;
+      b {
+        background: rgba(0, 0, 0, 0.1);
+        border-radius: 4px;
+        padding: 1px 4px;
+        font-size: 0.9em;
+        font-weight: 600;
+      }
+    }
+    &__date {
+      position: absolute;
```

Comment:
This is an absolute in a relative container, allowing the content of the item to be centered vertically while having the date always sticked in the corner. this is supposed to work just fine on mobile too